src/components/ebay-filter-menu/test/test.browser.js

```diff
@@ -0,0 +1,132 @@
+            expect(changeEvents).to.have.length(2);
+
+            const firstEventData = changeEvents[0][0];
+            const secondEventData = changeEvents[1][0];
+            expect(firstEventData).has.property('el', firstItem);
+            expect(secondEventData).has.property('el', secondItem);
+        });
+
+        it('then both items are selected', () => {
+            expect(firstItem).to.have.attr('aria-checked', 'true');
+            expect(secondItem).to.have.attr('aria-checked', 'true');
+        });
+    });
+
+    describe('when an item is checked and then unchecked', () => {
+        beforeEach(async() => {
+            await fireEvent.click(firstItem);
+            await fireEvent.click(firstItem);
+        });
+
+        it('then it emits the filter-menu-change events with correct data', () => {
+            const changeEvents = component.emitted('filter-menu-change');
+            expect(changeEvents).to.have.length(2);
+
+            const firstEventData = changeEvents[0][0];
+            const secondEventData = changeEvents[1][0];
+            expect(firstEventData).has.property('checked').to.deep.equal(['item 0']);
+            expect(secondEventData).has.property('checked').to.deep.equal([]);
+        });
+
+        it('then the item is unchecked', () => {
+            expect(firstItem).to.have.attr('aria-checked', 'false');
+        });
+    });
+
+    describe('when an item is checked via the keyboard', () => {
+        beforeEach(async() => {
+            await pressKey(firstItem, {
+                key: '(Space character)',
+                keyCode: 32
+            });
+        });
+
+        it('then it emits the filter-menu-change events with correct data', () => {
+            const changeEvents = component.emitted('filter-menu-change');
+            expect(changeEvents).to.have.length(1);
+
+            const firstEventData = changeEvents[0][0];
+            expect(firstEventData).has.property('el').with.text(firstItemText);
+            expect(firstEventData).has.property('checked').to.deep.equal(['item 0']);
+        });
+    });
+
+    describe('when the footer button is clicked', () => {
+        beforeEach(async() => {
+            await fireEvent.click(footerButton);
+        });
+
+        it('then it emits the filter-menu-footer-click event', () => {
+            const changeEvents = component.emitted('filter-menu-footer-click');
+            expect(changeEvents).to.have.length(1);
+        });
+    });
+
+    describe('when an item is added via input from its parent and the new item is clicked', () => {
+        const newInput = mock.Basic_3Items;
+        const thirdItemText = newInput.items[2].renderBody.text;
+        beforeEach(async() => {
+            await component.rerender(newInput);
+            await fireEvent.click(component.getByText(thirdItemText));
+        });
+
+        it('then the new item is selected or something');
```

Comment:
This probably shouldn't make it in right?

Reply:
Correct. Removing.